Makefile

```diff
@@ -0,0 +1,7 @@
+compile:
```

Comment:
Do we really need a makefile for docker commands? These can easily go into our README and the bonus is that people that don't know docker learn a bit about it.

Reply:
will add the docker commands to the readme. the makefile was to make the commands much easier to remember and much quicker to get up and running